docs/en/reference/sql/deployment_manage/ONLINE_SERVING_REQUIREMENTS.md

```diff
@@ -1,72 +1,76 @@
-# SQL On-Line Specifications and Requirements
+# Online Specifications and Requirements for SQL
```

Comment:
How about the file name? still serving

Reply:
the file name, corresponding index.rst files and links have been changed.
thx for reminding me~